src/main/java/org/tikv/common/TiSession.java

```diff
@@ -159,23 +159,46 @@ private static VersionInfo getVersionInfo() {
 
   private synchronized void warmUp() {
     long warmUpStartTime = System.currentTimeMillis();
+    warmUpStore();
+    warmUpRegion();
+    warmUpGrpc();
+    logger.info(
+        String.format("warm up duration %d ms", System.currentTimeMillis() - warmUpStartTime));
+  }
+
+  private void warmUpStore() {
     try {
+      BackOffer backOffer = ConcreteBackOffer.newRawKVBackOff();
       this.client = getPDClient();
       this.regionManager = getRegionManager();
-      List<Metapb.Store> stores = this.client.getAllStores(ConcreteBackOffer.newGetBackOff());
+      List<Metapb.Store> stores = this.client.getAllStores(backOffer);
       // warm up store cache
       for (Metapb.Store store : stores) {
         this.regionManager.updateStore(
-            null,
-            new TiStore(this.client.getStore(ConcreteBackOffer.newGetBackOff(), store.getId())));
+            null, new TiStore(this.client.getStore(backOffer, store.getId())));
       }
-      ByteString startKey = ByteString.EMPTY;
+    } catch (Exception e) {
+      // ignore error
+      logger.info("warm up store fails, ignored ", e);
```

Comment:
since the log is printed, it's better to print the failure reason from the exception to help to locate potential issues. warmup failures can be ignored by users, but cannot be ignored by us.

Reply:
the current code will print the `e` in log file.